src/constants.js

```diff
@@ -74,4 +74,4 @@ export const OPC_SIGNATURE = 'signature'
 export const OPC_SEED = 'seed'
 export const OPC_CONTRACT = 'contract'
 export const SHOW_UNSUPPORTED_FUNCTION = false
-
+export const SUPERNODES = ['ARMRp9R6RiFe771WPxEh8RMzNnhYssxZSKE', 'AR86RMV3YiVoMDH6JdqgQWzPVu4Rdpjo7B4', 'ARFW9By8BkDuNdnC1M4AbRN1DV4u6AMWWSw', 'ARE4NmwpsFYb1gkUnzATHQqwB6GoG4mmYS5', 'AR2qU3zyWx3TAd97W2LvRgBVmB2Hmz5285R', 'ARAyzTJewPDkTy2SgoS4GAUc6Y6ugKpL5uu', 'ARA8eM6VzLiF2PYtEbzDukB2JM6yDiA2yhj', 'ARPpk1ocPfRETUrAxqUPo5XYt6UBs9ct5in', 'AR7ns4xQ22fbTqxn9nTG4ijYPyiVWcKwuar', 'ARCXQ8R4a4B84cdcf6BrM4fXJ8SjhEM5hZG', 'ARCVpcq2i6rQ7kkzNeJ1jsMec6TLmC7RNHn', 'ARQNgdvu82J5hMF2UiKKoygxcH8p3KbRPm7', 'ARH7qKfAfqCwFsBC34exzBGa2iTZNKrFbhy', 'ARBQTCYws5FZAVtA1ZFLsGhBtPymr4Hp5CX', 'ARKWVGHcJGLaqTDqTq2zfmHWbc9EwRyBVB5', 'ARCsMvz24s2rtV3M5ARkJVkVgjHbY8GEPtG', 'AR85PjmwmJqK69VKWcXofyjSZgahsptZfMS', 'AR8mi8btwebYd73G3Kwv4ACYyZBfDMJvedx', 'ARB7u11TLc95q3T5ZD6EcMZADsDnSqGA1un', 'ARHyd2kRgqkRQsGkCBBzW6oF2fhYChpdN88', 'AR9QS36E61GCE1MHBhLXxNcgmD2ZftALGhf', 'ARMbfef7FWYQ1ebnYpVNZzfK5FKJ44eS3u5', 'ARB9MqZJ1aLGbEQDd5ZfhUmjggcS6tKwTPK', 'ARDeXgTQW1i87Gq6wGvkUP3qt3JhzDHUEtr', 'ARRGpkqC2is1kvCMNvRftZtSMbx2PrttRoC', 'AR4Exapc1LnZ5nYkGwPvmpD7QZtEsdRQoGk', 'ARHvo4EEJjbupPBYFgf3EVrsR9aMYYRKf9G', 'ARLfqnrjuBzHRU2AwpVtY9hfEU54LUC5BYd', 'ARF51tsoLsTQ8PMyf6jyWnJChdDR3RYkR2J', 'ARGuxwy6sEHVhR2eu68gsbZ7VE98tQb7TjN', 'ARCug95rfqzbSRuYhREudLNVdhu9i2X97cH']
```

Comment:
The supernode address will be changed dynamically. It should not put supernode address in config. The supernode addresses should retrieve from GET /consensus/allSlotsInfo API.